ios/src/platform_ios.mm

```diff
@@ -126,14 +132,115 @@ bool isContinuousRendering() {
     return ptr;
 }
 
-// No system fonts implementation (yet!)
-std::string systemFontPath(const std::string& _name, const std::string& _weight, const std::string& _face) {
-    return "";
+unsigned char* loadUIFont(UIFont* _font, size_t* _size) {
+
+    CGFontRef fontRef = CGFontCreateWithFontName((CFStringRef)_font.fontName);
+
+    if (!fontRef) {
+        *_size = 0;
+        return nullptr;
+    }
+
+    unsigned char* data = [CGFontConverter fontDataForCGFont:fontRef size:_size];
+
+    CGFontRelease(fontRef);
+
+    if (!data) {
+        LOG("CoreGraphics font failed to decode");
+
+        *_size = 0;
+        return nullptr;
+    }
+
+    return data;
+}
+
+std::vector<FontSourceHandle> systemFontFallbacksHandle() {
+    NSArray* fallbacks = [UIFont familyNames];
+
+    std::vector<FontSourceHandle> handles;
+
+    for (id fallback in fallbacks) {
```

Comment:
There a reason this is an `id` type here? https://developer.apple.com/reference/uikit/uifont/2091412-familynames?language=objc

Seems to indicate objects inside the return array will be of type `NSString`.